test/workload_discovery/windows/util/scripts.ps1

```diff
@@ -1,27 +1,19 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT
 
+# (TODO: remove function once race conditions between conflicting java processes is resolved)
+function Terminate-Java {
+    Get-Process -Name "java" -ErrorAction SilentlyContinue | Stop-Process -Force
+}
```

Comment:
This doesn't seem like a great solution. Why not terminate specific Java processes if you need to terminate them at all?